src/Symfony/Component/HttpKernel/Resources/welcome.html.php

```diff
@@ -19,8 +19,9 @@
-        .warning { background: <?= $lightColor(); ?>; display: flex; align-content: center; padding: 10px; text-align: left; justify-content: center; }
-        .warning svg { height: 48px; width: 48px; margin-right: 10px; }
+        .warning { background: <?= $lightColor(); ?>; display: flex; align-content: center; align-items: center; padding: 10px; text-align: left; justify-content: center; }
```

Comment:
`align-content` is actually useless. It has no effect in Flexbox when you don't enable `flex-wrap: wrap`

Reply:
You are right! It's removed now. Thanks.